cumulusci/tasks/tests/test_datadictionary.py

```diff
@@ -910,7 +912,9 @@ def test_run_task__additional_dependencies(self):
         task._get_repo_dependencies = Mock(return_value=[1, 2])
         task._walk_releases = Mock()
 
-        task._run_task()
+        with TemporaryDirectory() as t:
+            with cd(t):
+                task._run_task()
```

Comment:
I'd suggest using cumulusci.utils.temporary_dir which creates a dir _and_ cds to it.